src/libs/Log.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable rulesdir/no-api-in-views */
```

Comment:
It needs to be instead an "actions" file. I could fix this. Basically, we would need to create an actions file for `Log` that is just a wrapper for `API.Log()`.

Reply:
I think I won't though because it will be super confusing and people won't know whether to use the "action" or the `Log` instance (and they shouldn't use the action).

Reply:
Sounds good, thanks for the explanation.